webpack/scenes/ContentViews/Create/CreateContentViewFormComponents.js

```diff
@@ -0,0 +1,70 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+import React from 'react';
+import { Popover, Flex, FlexItem } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+import ContentViewIcon from '../components/ContentViewIcon';
+
+
+export const labelComposite = () => (
```

Comment:
A function that returns JSX is a React component, so the name should be styled as such: `LabelComposite`